DataProc/CPass1/main_recCPass1.C

```diff
@@ -105,7 +105,11 @@ void main_recCPass1(const char *filename="raw.root",Int_t nevents=-1, const char
   }
 
   // All friends
-  rec.SetFractionFriends(.2);
+  TString collSystem(strcmp(gSystem->Getenv("ALIEN_JDL_LPMINTERACTIONTYPE")));
```

Comment:
Ciao Chiara,

Looks good, except for the call for strcmp... 
Could you please remove it?
Thanks!

Yours,
Catalin.

Reply:
You are right!!! Sorry :-)

Reply:
Ciao Chiara,

Now it's perfect, thanks a lot!

Yours,
Catalin.